src/content/docs/style-guide/images/screenshots-images.mdx

```diff
@@ -246,35 +253,6 @@ If you're relying on surrounding text to explain your image:
 </CollapserGroup>
 
 
-### Image annotations [#annotations]
-
-Image annotations are text and shapes in screen captures or diagrams that help to explain the image. If an image has annotations that explain what readers need to know, captions may be redundant.
-
-When using annotations:
-
-* Be concise.
-* Make sure annotations are easy to distinguish from UI text. For example, you may need to offset your boxes so they don't look like UI elements.
-* Annotations aren't translated, so use them sparingly.
-* Use our [New Relic SnagIt theme](https://github.com/newrelic/newrelic-doc-tools) (link only available to New Relic employees) to keep your annotations consistent. To import the New Relic theme:
-  1. Open SnagIt.
-  2. In the **Quick Styles** pane, select the **Theme** gear icon.
-  3. Select **Import**.
-
-<CollapserGroup>
-  <Collapser
-    id="annotations-example"
-    title="Annotations as titles"
-  >
-    Here's an example of an image that has an annotation at the top that acts like a title:
-
-    <img
-      title="Annotations_example.png"
-      alt="Diagram that shows an example of annotations."
-      src={styleguideDocsDiagramWithHardcodedTitleExample}
-    />
-  </Collapser>
-</CollapserGroup>
-
 ## Delete or rename an image file [#delete-rename-image]
 
 Because our translated mdx files in the i18n folder reference the same image folder that our English-language mdx files do, when image files are deleted or renamed, this can throw a build error because of the mismatch between the English files and the translated files. Here are some tips for deleting or renaming an image file: 
```

Comment:
```suggestion
Our i18n translated .mdx files need to match the corresponding English .mdx files. When you delete or rename image files in English, you might cause a build error because there's now a mismatch between the English and translated files. Here are some tips when deleting or renaming an image file: 
```

Reply:
Added the modification! Thanks!